libs/net/services/Config/ServiceOptions.cs

```diff
@@ -27,6 +27,6 @@ public class ServiceOptions
     /// <summary>
     /// get/set - The service timezone.
     /// </summary>
-    public string TimeZone { get; set; } = "Pacific Standard Time";
+    public string TimeZone { get; set; } = "UTC";
```

Comment:
The system time is normally UTC.  Hoping this fix for Syndication Service doesn't break the Capture and Clip Services.